src/UglyToad.PdfPig/Writer/Copier/TokenHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace UglyToad.PdfPig.Writer.Copier
+{
+    using System;
+    using System.IO;
+    using Core;
+    using Tokens;
+
+    internal static class TokenHelper
```

Comment:
I thought about using the `DirectObjectFinder` helper class that we have, but I think this method offer more flexibility in term of what can be a token scanner